tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_with_toy_connectivity.py

```diff
@@ -452,6 +464,8 @@ def test_sparse_shifted_stencil_reduce(program_processor_no_gtfn_exec, lift_mode
     if program_processor == gtfn.format_sourcecode:
         pytest.xfail("We cannot unroll a reduction on a sparse field only.")
         # With our current understanding, this iterator IR program is illegal, however we might want to fix it and therefore keep the test for now.
+    if program_processor == run_dace_iterator:
+        pytest.xfail("Not supported in DaCe backend: assert fails")
```

Comment:
```suggestion
        pytest.xfail("Illegal iterator IR")
```
I understand that you don't want to create a joint skip gtfn and dace fixture for this case, but maybe we can improve the error message.